downloader/build.gradle

```diff
@@ -4,11 +4,52 @@ apply plugin: 'application'
 applicationName = 'downloader'
 mainClassName = 'com.here.ort.downloader.Main'
 
+sourceSets {
+    funTest {
+        kotlin.srcDirs 'src/funTest/kotlin'
+    }
+}
+
 dependencies {
     compile project(':model')
     compile project(':util')
 
     compile "com.beust:jcommander:$jcommanderVersion"
 
     testCompile "io.kotlintest:kotlintest:$kotlintestVersion"
+
+    funTestCompile sourceSets.main.output
+    funTestCompile sourceSets.test.output
+    funTestCompile configurations.testCompile
+    funTestRuntime configurations.testRuntime
+
+    funTestCompile "io.kotlintest:kotlintest:$kotlintestVersion"
+
 }
+
+task funTest(type: Test) {
+    description = 'Runs the functional tests.'
+    group = 'Verification'
+    classpath = sourceSets.funTest.runtimeClasspath
+    testClassesDirs = sourceSets.funTest.output.classesDirs
+
```

Comment:
This now also needs the system property forwarding of https://github.com/heremaps/oss-review-toolkit/pull/84/files#diff-92b3f5522dfb0d3329dd9fdda5323737R44